spec/features/work_show_spec.rb

```diff
@@ -55,7 +55,7 @@
     end
 
     it "allows adding work to a collection", clean_repo: true, js: true do
-      optional 'ability to get capybara to find css select2-result (see Issue #3038)' if ci_build?
+      # optional 'ability to get capybara to find css select2-result (see Issue #3038)' if ci_build?
```

Comment:
Can you elaborate on why this change and the addition of `sleep 5` below are needed?

Reply:
After a save is issued, there can be some delay as it processes before the object is available for the subsequent commands. If you debug, it slows things down and works, but when run without debug, it fails. Sleep allows the page to be fully available for the subsequent tests.

Reply:
okay to just remove this line, rather than commenting?

Reply:
I left the comment temporarily, to keep the reference to the related issue until I was sure it fixed the problem.  I intended to remove it later, but if this PR has other changes needed, I'll remove it here.

Reply:
My worry would be that these kinds of comments can end up being out of sight, out of mind and never get deleted until someone stumbles across it.  And that person may not be sure it is ok to remove.  

Perhps you could add a comment referencing the issue near the `sleep` statement.  

_Caveat: I'm not familiar with `optional` in rspec tests and a quick google search did not enlighten me.  So my suggestion may not make sense if I guessed its function incorrectly._

Something like...
```
# This attempts to address issue #3038.  Previously, this test was marked optional.

Reply:
I could, but honestly, I intend to remove the comment entirely in my next PR. I just wanted to leave it when I pushed up the code, to make sure it fixed it on Travis as well as locally, so it would be easier to find if it didn't work. Is it worth removing it or adding a different comment in this PR? or should i just do it as part of my next one?

Reply:
That seems fine if you are doing it in the next PR.